extensions/amp-social-share/0.1/amp-social-share.js

```diff
@@ -184,19 +195,6 @@ class AmpSocialShare extends AMP.BaseElement {
       openWindowDialog(this.win, href, target, windowFeatures);
     }
   }
-
-  /**
-   * @private
-   * @return {*} TODO(#23582): Specify return type
-   */
-  systemShareSupported_() {
-    // Chrome exports navigator.share in WebView but does not implement it.
-    // See https://bugs.chromium.org/p/chromium/issues/detail?id=765923
-    const isChromeWebview =
-      this.viewer_.isWebviewEmbedded() && this.platform_.isChrome();
```

Comment:
Was the chrome webview bug fixed?

Reply:
The original bug is [no longer reproducible ](https://bugs.chromium.org/p/chromium/issues/detail?id=765923#c34) (Chrome 66 +) even though it's not marked fixed. Alan and I discussed that it would be safe to remove the workaround for it.